webui/src/app/charts/line-chart/line-chart.component.ts

```diff
@@ -52,9 +54,22 @@ export class LineChartComponent implements OnChanges, OnInit {
   margin = { top: 40, right: 40, bottom: 60, left: 60 };
   loading = true;
 
+  private readonly resize$$ = this.windowService.resizeDebounce$.subscribe(
+    w => {
```

Comment:
```suggestion
    () => {
```

Reply:
Good catch, will remove this unused parameter.